app/services/vacancy_algolia_search_builder.rb

```diff
@@ -0,0 +1,106 @@
+require 'geocoding'
+
+class VacancyAlgoliaSearchBuilder
+  include ActiveModel::Model
+
+  attr_accessor :keyword, :location_category, :location, :radius, :sort_by, :page,
+                :search_query, :location_filter, :search_replica, :search_filter,
+                :vacancies
+
+  DEFAULT_RADIUS = 10
+
+  # This is so I can set the class variable from outside the class. Must be a better way to do this
```

Comment:
Maybe I'm missing something, but `call` is an instance method, so it doesn't need `default_hit_per_page` to be a class attribute. Make it an instance variable that can be overridden in the initializer: 

```
DEFAULT_PER_PAGE = 10

attr_accesssor ..., :per_page 

def initialize(params)
...
 
  self.per_page = params.delete(:per_page) || DEFAULT_PER_PAGE
 
...

def call 
  self.vacancies = Vacancy.search(
    ...
    hitsPerPage: per_page,
    ...
```
...

...